css/style.css

```diff
@@ -1,5 +1,92 @@
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Inter:wght@400;500;700&family=Poppins&family=Roboto:wght@300;400;700&display=swap');
+
+/* Primary colours */
+:root {
+  --black: #1c1a19;
+  --orange: #ff6b00;
+  --off_white: #fff5e1;
+  --white: #fff;
+  --grey: #3c3a39;
+  --light_gray: #dbd8d7;
+  --nav_height: 45px;
+}
+
 *,
 *::before,
 *::after {
   box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+a {
+  text-decoration: none;
+  color: var(--white);
+}
+
+header {
+  color: white;
+}
+
+.toolbar {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 24px 0 24px;
+  background: var(--grey);
+  font-family: 'Roboto', sans-serif;
+  height: var(--nav_height);
+}
+
+nav h1 {
+  font-size: 18px;
+  color: var(--white);
+}
+
+.headline-container h1 span {
+  display: block;
+}
+
+.headline-container {
+  background: var(--black);
+  background-image: url("../assets/img/header_mobile.png");
+  background-position: right bottom; /* Positioning */
+  background-size: 100px;
+  background-repeat: no-repeat; /* Prevent showing multiple background images */
+  height: calc(100vh - var(--nav_height));
+  padding: 80px 24px 0 24px;
+}
+
+.headline h1 span {
+  font-family: 'Crete Round', serif;
+  font-style: normal;
+  font-size: 50px;
+  letter-spacing: 0.37px;
+  line-height: 64px;
+  font-weight: 400;
+  color: var(--orange);
+}
+
+.headline h2 {
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 24px;
+  line-height: 32px;
+  color: var(--off_white);
+}
+
+.headline p {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  margin: 10px 0 20px 0;
+}
+
+.socials i {
+  margin-right: 15px;
+  color: var(--light_gray);
+  font-size: 20px;
 }
```

Comment:
- [ ] The `color: var` style is actually generating a bug on the social media icons.
### **Take a look below**
![confuse](https://user-images.githubusercontent.com/55596970/158944646-07235272-a5f0-4b6e-a5bc-2746cb40de74.png)

Reply:
Thank you @CollinsTatang for the kind feedback. I'm thinking the icons didn't display properly because they were on a white background which stemmed from the background image bug. I have fixed the background image bug.

![image](https://user-images.githubusercontent.com/20554583/158962029-6bbb9fd0-3d4d-4f33-a824-a4d0a2a89c89.png)


Check again and let me know if there's more to it.